contracts/strategies/strategy.sol.example

```diff
@@ -0,0 +1,125 @@
+/// @notice SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+/// @dev Import the appropriate base contract
+import "../strategy-joe-rush-farm-base.sol";
+
+/// @dev Contracts use NatSpec Format comments. NatSpec documentation is located here: https://docs.soliditylang.org/en/develop/natspec-format.html
+/// @notice The strategy contract for TraderJoe's Token1/Token2 Liquidity Pool with double rewards
+/// @dev If one of the tokens is a bridge extension token (ends with ".e"), the trailing E should be capitalized like so: StrategyJoeAvaxDaiE
+contract StrategyJoeToken1Token2 is StrategyJoeRushFarmBase {
+    /// @dev LP and Token addresses
+    uint256 public token1_token2_poolId = 10;
```

Comment:
State variables need to be in UpperCamelCase in order to differentiate them from contractMethods(), localVariables and _functionParameters.

Additionally, all variables should ideally be named in a way which prevents an inline comment from being required in order to explain what they are.

Reply:
Update: I see in our documentation that it says to use snake_case for state variables that are basic types.  We should discuss with the team if it makes sense to continue this way, or if this needlessly complicates things.  If the variables are well-named, the type should be either implicit or explicit in the name.  So if we named this LiquidityPoolID, that would be implicitly stating that it's a number.